app/views/News.js

```diff
@@ -55,7 +55,8 @@ class NewsScreen extends Component {
 
   renderItem = item => {
     return (
-      <View style={styles.singleNews}>
+      // <View style={styles.singleNews}>
```

Comment:
remove dead code

remove the unused style block below

However, I think you want `alignItems: 'stretch'` here.